src/sentry/sentry_metrics/indexer/postgres.py

```diff
@@ -33,8 +35,21 @@ def bulk_record(self, strings: List[str]) -> Mapping[str, int]:
 
         mapped_result: MutableMapping[str, int] = {r.string: r.id for r in cache_results}
 
+        metrics.incr("sentry_metrics.indexer.memcache.fetch", amount=len(strings))
         unmapped = set(strings).difference(mapped_result.keys())
-        new_mapped = self._bulk_record(unmapped)
+        if not unmapped:
+            # This will probably be very rare in practice since for each batch of strings
+            # it's almost certain there would be a value we haven't seen before
+            metrics.incr("sentry_metrics.indexer.memcache.hit", amount=len(strings))
```

Comment:
Let's extract these strings (`sentry_metrics.indexer.memcache.hit`) to constants so that if we want to change them then they only need to be changed in one place